os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_common/ameba_ota.c

```diff
@@ -430,4 +430,28 @@ int OTA_KernelImageSignatureCheck(uint32_t input_addr)
 	return TRUE;
 }
 
+int Ameba_KeyDeriveFunc(const unsigned char *password, size_t plen,
```

Comment:
Could you leave any description about return value?

Reply:
Dear @sunghan-chang ,
I have added description for this API.